### ISSUE TITLE
Add nixosTest

### ISSUE THREAD

**Issue by srid** [juspay/jenkins-nix-ci#28]
Should spin up Jenkins master and make sure it runs

**Comment by con-f-use** [comment]
Which is probably blocked by the fact that a [major component is broken](https://github.com/Fuuzetsu/jenkinsPlugins2nix/issues/12).

**Comment by bjornfor** [comment]
It's been a while since I looked at it, but I was thinking we should make https://github.com/jenkinsci/plugin-installation-manager-tool emit JSON metadata for plugins that can be imported by a small Nix lib.

**Comment by con-f-use** [comment]
Yeah sure, but it does not bode well for future reliability and isn't great for usability.
